src/main/java/frc/robot/Constants.java

```diff
@@ -19,11 +24,87 @@ public class Constants {
     public static final double MK4I_STEER_INERTIA = 0.00001;
     public static final double MK4I_STEER_FRICTION_VOLTAGE = 0.25;
     public static final double MK4_WHEEL_RADIUS = Units.inchesToMeters(1.93);
+    public static final double WHEEL_COF = 1.2;
     public static final Slot0Configs MK4I_STEER_PID_CONSTANTS = new Slot0Configs()
             .withKP(50);
 
     public static final double DEBOUNCE_TIME = 0.5;
 
+    public static final double yOffset = 158.5;
+    public static final double xBranchDistanceFromCenter = 31.625;
+    public static final double yBranchDistanceFromCenter = 6.5;
```

Comment:
Make sure to follow the correct naming convention for constants. Also be more specific so it is known that these are referring to the reef. These should also be converted to meters here, instead of in the getBranchPose method.